server/runtime/src/main/java/org/infinispan/server/configuration/ServerConfigurationParser.java

```diff
@@ -905,6 +928,28 @@ private void parsePropertiesRealm(ConfigurationReader reader, PropertiesRealmCon
       propertiesBuilder.build();
    }
 
+   private void parseDistributedRealm(ConfigurationReader reader, DistributedRealmConfigurationBuilder distributedRealmBuilder) {
+      String name = "distributed";
+      String[] realms = ParseUtils.requireAttributes(reader, Attribute.REALMS)[0].split("\\s+");
```

Comment:
nit
```suggestion
      String[] realms = ParseUtils.requireSingleAttribute(reader, Attribute.REALMS).split("\\s+");
```

Reply:
We cannot do that, since we also want to parse `NAME` optionally. `requireSingleAttribute` will fail if the attribute count is `!=1`